microcom.c

```diff
@@ -144,54 +147,57 @@ int main(int argc, char *argv[])
 		{ },
 	};
 
-	while ((opt = getopt_long(argc, argv, "hp:s:t:c:dfl:oi:a:e:v", long_options, NULL)) != -1) {
+	while ((opt = getopt_long(argc, argv, "hp:s:t:c:dqfl:oi:a:e:v", long_options, NULL)) != -1) {
 		switch (opt) {
-			case '?':
-				main_usage(1, "", "");
-				break;
-			case 'h':
-				main_usage(0, "", "");
-				break;
-			case 'v':
-				printf("%s\n", PACKAGE_VERSION);
-				exit(EXIT_SUCCESS);
-				break;
-			case 'p':
-				device = optarg;
-				break;
-			case 's':
-				current_speed = strtoul(optarg, NULL, 0);
-				break;
-			case 't':
-				telnet = 1;
-				hostport = optarg;
-				break;
-			case 'c':
-				can = 1;
-				interfaceid = optarg;
-				break;
-			case 'f':
-				opt_force = 1;
-				break;
-			case 'd':
-				debug = 1;
-				break;
-			case 'l':
-				logfile = optarg;
-				break;
-			case 'o':
-				listenonly = 1;
-				break;
-			case 'a':
-				answerback = optarg;
-				break;
-			case 'e':
-				if (strlen(optarg) != 1) {
-					fprintf(stderr, "Option -e requires a single character argument.\n");
-					exit(EXIT_FAILURE);
-				}
-				escape_char = *optarg;
-				break;
+		case '?':
+			main_usage(1, "", "");
+			break;
+		case 'h':
+			main_usage(0, "", "");
+			break;
+		case 'v':
+			printf("%s\n", PACKAGE_VERSION);
+			exit(EXIT_SUCCESS);
+			break;
+		case 'p':
+			device = optarg;
+			break;
+		case 's':
+			current_speed = strtoul(optarg, NULL, 0);
+			break;
+		case 't':
+			telnet = 1;
+			hostport = optarg;
+			break;
+		case 'c':
+			can = 1;
+			interfaceid = optarg;
+			break;
+		case 'f':
+			opt_force = 1;
+			break;
+		case 'd':
+			debug = 1;
+			break;
+		case 'q':
+			quiet = 1;
+			break;
+		case 'l':
+			logfile = optarg;
+			break;
+		case 'o':
+			listenonly = 1;
+			break;
+		case 'a':
+			answerback = optarg;
+			break;
+		case 'e':
+			if (strlen(optarg) != 1) {
+				fprintf(stderr, "Option -e requires a single character argument.\n");
+				exit(EXIT_FAILURE);
+			}
+			escape_char = *optarg;
+			break;
```

Comment:
This mixes a formatting change with a code change. It would be great to only have the

```diff
+		case 'q':
+			quiet = 1;
+			break;
```

In this commit.